package.json

```diff
@@ -19,14 +19,14 @@
 		"rimraf": "^2.6.1",
 		"symlink-or-copy": "^1.2.0"
 	},
-	"babel": {
-		"presets": "@wordpress/default"
-	},
```

Comment:
Why are we removing this, is it useless?

Reply:
Jest can read Babel from its config, and the script which generates the distribution folders for packages uses the same preset internally. So it isn't necessary anymore - it uses defaults :)

Reply:
I had to add `babel.config.js` instead to make everything work 😄